pynest/examples/CampbellSiegert.py

```diff
@@ -204,11 +202,11 @@ def psp(x):
 # omitted so initial transients do not perturb our results. We then print the
```

Comment:
Above, I'd suggest to change
`pg = nest.Create('poisson_generator', len(rates),
                 [{'rate': float(rate_i)} for rate_i in rates])`
to
`pg = nest.Create('poisson_generator', len(rates), {'rate': rates})`

Reply:
done